environment-dev.yml

```diff
@@ -0,0 +1,65 @@
+name: eventkit-cloud
+channels:
+ - local
+dependencies:
+ - python=2.7.15
+ - django-audit-logging=0.1.9
+ - gdal=2.2.2-0
+ - geos=3.6.1
+ - libgdal=2.2.2-0
+ - osmctools=0.7.0
+ - openldap=2.4.45
+ - pip=9.0.1
+ - proj4=4.9.3
+ - python-ldap=2.4.39
+ - pip:
```

Comment:
Is it OK to still have pip? You could move all of this into Conda if you wanted to. Pip is still going to reach out to the open internet, right?

Reply:
There are some recipes that aren't yet available.  It is try that pip does reach out to the open internet.   Cloud Foundry will still let you vendor those dependencies separately.  A subsequent PR will provide recipes for the conda dependencies that aren't available in any public repository.